src/javascript/app_2/App/Components/Layout/Header/login_button.jsx

```diff
@@ -0,0 +1,19 @@
+import PropTypes    from 'prop-types';
+import React        from 'react';
+import Button       from '../../Form/button.jsx';
+import { localize } from '../../../../../_common/localize';
+
+const LoginButton = ({ onClick }) => (
+    <Button
+        className='primary orange'
+        has_effect
+        text={localize('Log in')}
+        onClick={onClick}
```

Comment:
Can we use `redirectToLogin` here like before?

Reply:
should we import `redirectToLogin` here or pass in from Container [here](https://github.com/binary-com/binary-static/pull/4118/files/703e5c68a9de5cad33a2f20b8c243c8da018735d#diff-7879ff98b17472fff8f94105aab44427R42) ?

Reply:
We can import it here, as it is always the same for logging in.

Reply:
ok thanks :D